openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceMetadataPipeline.json

```diff
@@ -23,8 +23,8 @@
       "type": "boolean",
       "default": true
     },
-    "markDeletedTablesFromFilterOnly": {
-      "description": "Optional configuration to mark deleted tables only to the filtered schema",
+    "markAllDeletedTables": {
```

Comment:
If we rename a JSON field that we are storing in DB, we must add it to migration files.

Reply:
Done